requests.http

```diff
@@ -1,6 +1,6 @@
 ### Generate bikes

```

Comment:
Maybe it's better to leave the values as they were?

Reply:
Yeap, will do, this is a local change I did to run locally and it leaked to the commit...